.circleci/config.yml

```diff
@@ -403,6 +415,28 @@ jobs:
           coverage-reports: /tmp/workspace/ui/coverage/lcov.info
           project-token: $CODACY_PROJECT_TOKEN
 
+  api-contract-upload:
+    executor: pact-executor
```

Comment:
Using a different executor, because it has the pact-broker cli installed on it, and that is the main thing that I need for these tasks.